src/release/Release.js

```diff
@@ -26,29 +27,29 @@ export default class Release {
   }
 
   async owner() {
-    return await this._release.owner(this.txParams)
```

Comment:
Does removing `this.txParams` here have anything to do with gas estimation? Otherwise, why remove it?

Reply:
I preferred to remove the `txParams`, which have information for transactions (such as sender), from all the static calls. They should not make any difference at all.